src/transformState.js

```diff
@@ -5,7 +5,33 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const TYPE_ADD_PROPERTIES = 'addProperties';
+  const TYPE_REMOVE_PROPERTIES = 'removeProperties';
+  const TYPE_CLEAR = 'clear';
```

Comment:
You don't need to create new variables. You can use 'addProperties', 'removeProperties',  'clear'.  We need to create new variables to describe value instead hardcoding. It's not such case.